### PR TITLE
acronyms and compact

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: acronyms.tex [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
 DM & Data Management \\\hline
 DMTN & DM Technical Note \\\hline
 DRP & Data Release Production \\\hline
-Data Release Production & An episode of (re)processing all of the accumulated LSST images, during which all output DR data products are generated. These episodes are planned to occur annually during the LSST survey, and the processing will be executed at the Archive Center. This includes Difference Imaging Analysis, generating deep Coadd Images, Source detection and association, creating Object and Solar System Object catalogs, and related metadata. \\\hline
 FITS & Flexible Image Transport System \\\hline
 GB & Gigabyte \\\hline
-GCS & Google Cloud Storage \\\hline
+GCE & Google Cloud Engine \\\hline
+GCP & Google Cloud Platform \\\hline
+GCS & Generic Control System \\\hline
```
Comment (ktlim) [inline acronyms.tex]: Original was correct.
```suggestion
GCS & Google Cloud Storage \\\hline
```

File: acronyms.tex [hunk 2]
```diff
@@ -1,48 +1,41 @@
...
 ComCam & The commissioning camera is a single-raft, 9-CCD camera that will be installed in LSST during commissioning, before the final camera is ready. \\\hline
 DAQ & Data Acquisition System \\\hline
 DM & Data Management \\\hline
 DMTN & DM Technical Note \\\hline
 DRP & Data Release Production \\\hline
-Data Release Production & An episode of (re)processing all of the accumulated LSST images, during which all output DR data products are generated. These episodes are planned to occur annually during the LSST survey, and the processing will be executed at the Archive Center. This includes Difference Imaging Analysis, generating deep Coadd Images, Source detection and association, creating Object and Solar System Object catalogs, and related metadata. \\\hline
 FITS & Flexible Image Transport System \\\hline
 GB & Gigabyte \\\hline
-GCS & Google Cloud Storage \\\hline
+GCE & Google Cloud Engine \\\hline
```
Comment (ktlim) [inline acronyms.tex]:
```suggestion
GCE & Google Compute Engine \\\hline
```

File: acronyms.tex [hunk 3]
```diff
@@ -1,48 +1,41 @@
...
+GCP & Google Cloud Platform \\\hline
+GCS & Generic Control System \\\hline
 HSC & Hyper Suprime-Cam \\\hline
 HTTP & HyperText Transfer Protocol \\\hline
+IAM & Identity Access Management \\\hline
```
Comment (ktlim) [inline acronyms.tex]:
```suggestion
IAM & Identity and Access Management \\\hline
```

File: DMTN-157.tex [hunk 4]
```diff
@@ -44,16 +44,16 @@
...
+\mkshorttitle
 
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 \section{Introduction}
 
 In spring 2020, we started the second Proof of Concept (PoC) engagement with the Google Cloud team; see \citeds{DMTN-150} for the plans.
-Section \ref{sec:drp} reports on the batch Data Release Production (DRP) processing on Google Cloud Platform (GCP) and its cost.
+ \secref{sec:drp} reports on the batch Data Release Production (DRP) processing on Google Cloud Platform (GCP) and its cost.
 Section \ref{sec:dataxfer} reports on data transfer testing between Chile and Google Cloud Storage (GCS).
```
Comment (ktlim) [inline DMTN-157.tex]: Do the same substitution in the second use.
```suggestion
 \secref{sec:drp} reports on the batch Data Release Production (DRP) processing on Google Cloud Platform (GCP) and its cost.
\secref{sec:dataxfer} reports on data transfer testing between Chile and Google Cloud Storage (GCS).
```

**Review by ktlim — APPROVED** [review]

File: acronyms.tex (hunk 2 above)
Comment (womullan) [reply acronyms.tex]: ok - changed in main glossary

File: acronyms.tex (hunk 3 above)
Comment (womullan) [reply acronyms.tex]: got it

**Pull request merged**